src/main/kotlin/no/nav/medlemskap/domene/Medlemskap.kt

```diff
@@ -17,10 +17,22 @@ data class Medlemskap (
 
 }
 
-enum class DekningForSykepenger(val dekning: String) {
+enum class Dekning(val dekningKodeverdi: String) {
```

Comment:
MEDL har et kodeverk for dekning, du finner det her: https://kodeverk-web.nais.adeo.no/kodeverksoversikt/kodeverk/DekningMedl
Burde vi samkjørt oss med det på noen måte? Det ser egentlig ut som om du delvis har gjort det allerede?

Reply:
Jepp! Fikset.